src/bin/lttng-relayd/main.c

```diff
@@ -1527,7 +1527,7 @@ int create_rotate_index_file(struct relay_stream *stream)
 			stream->channel_name,
 			-1, -1, stream->tracefile_size,
 			tracefile_array_get_file_index_head(stream->tfa),
-			lttng_to_index_major(major, minor),
+			lttng_to_index_major(major),
```

Comment:
I'm not sure about this one... I think exposing a different index would not necessarily need a major version change... I'll have to double check.

Reply:
Fixed by https://github.com/lttng/lttng-tools/commit/5e943e75cbb2d63c95d674e523e078050f1c868c